// Copyright 2009 Google Inc. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Protocol buffer used to represent minimize dns results.

syntax = "proto2";

import "pagespeed/proto/pagespeed_output.proto";

package pagespeed;

message MinimizeDnsDetails {
  extend ResultDetails {
    // In order to attach it to Result.details
    optional MinimizeDnsDetails message_set_extension = 4484723;  // int('Dns')
  }

  // Number of hosts used to serve the page's content.
  optional int32 num_hosts = 1;

  // List of wasteful url requests
  repeated string violation_urls = 2;
}
